openedx/adg/lms/webinars/helpers.py

```diff
@@ -277,23 +277,32 @@ def send_webinar_invitation_emails(webinar, old_webinar_data=None):
     if invite_email_recipients:
         send_webinar_emails(MandrillClient.WEBINAR_CREATED, webinar, list(invite_email_recipients))
 
-    if unregistered_members_emails:
-        send_webinar_emails(MandrillClient.WEBINAR_TEAM_INVITE, webinar, list(unregistered_members_emails))
+    member_email_recipients_with_no_platform_account = unregistered_members_emails.intersection(updated_webinar_emails)
```

Comment:
I think `member_email_recipients_with_no_platform_account` is too long of a name for a variable, maybe we can rename it to something like `new_unregistered_members_emails` as we are extracting those emails that have been newly added and there is no platform account against those emails.